load-test-framework/src/main/java/com/google/pubsub/flic/controllers/resource_controllers/PubsubResourceController.java

```diff
@@ -58,21 +58,22 @@ protected void startAction() throws Exception {
           .execute();
     } catch (GoogleJsonResponseException e) {
       if (e.getStatusCode() != HttpStatus.SC_CONFLICT) {
-        log.error("Error creating subscription");
+        log.error("Error creating topic {}", topic);
         throw e;
       }
-      log.info("Topic already exists, reusing.");
+      log.info("Topic {} already exists, reusing.", topic);
     }
     for (String subscription : subscriptions) {
-      log.error("Creating subscription: " + subscription);
+      log.error("Creating subscription: {}", subscription);
       try {
         pubsub
             .projects()
             .subscriptions()
             .delete("projects/" + project + "/subscriptions/" + subscription)
             .execute();
       } catch (IOException e) {
-        log.debug("Error deleting subscription, assuming it has not yet been created.", e);
+        log.debug("Error deleting subscription {}, assuming it has not yet been created. {}",
```

Comment:
> assuming it has not yet been created.
This fragment is not needed because the error will show a NotFoundException to make clear the subscription we attempted to delete does not exist.

Reply:
thanks, removed that part.